### ISSUE TITLE
Since new v1.13 release "Get it now" button doesnt do anything

### ISSUE THREAD

**Issue by schroef** [adobe/brackets#14612]
v1.13.0-17969
osx 10.11.6

I noticed that a new release was available. Yet the button "Get it now" doesnt initiate the action it used to do. Normally im guided to the download page, now nothing happens.

The "Release Notes", does work but showes info from a different release i believe. I states that in this issue #14611 
<img width="824" alt="screen shot 2018-12-15 at 21 37 52" src="https://user-images.githubusercontent.com/6923008/50049086-8100c200-00b2-11e9-8177-bee39702d3f2.png">
<img width="628" alt="screen shot 2018-12-15 at 21 36 36" src="https://user-images.githubusercontent.com/6923008/50049087-81995880-00b2-11e9-99c3-877b238bb54b.png">

**Comment by xtoq** [comment]
This release is for Windows only. If you're on a Mac (you say you're on OSX so I assume so?), then there's nothing to update!

**Comment by schroef** [comment]
I understand, but now i see this update icon all the time.
<img width="59" alt="screen shot 2018-12-16 at 17 17 20" src="https://user-images.githubusercontent.com/6923008/50058946-7bf54e80-0156-11e9-83b4-2dd0ac5c97a0.png">

**Comment by badcat** [comment]
Also on macOS and seeing the update green icon.  Seems like this should not be happening and is certainly leading to confusion.

**Comment by Rainbowlemon** [comment]
IMO even if the update for other operating systems added nothing at all, it should still happen just to get rid of the 'new build available' icon.

**Comment by HarrisonRiddell** [comment]
Agreed. I work on both mac and windows and I was able to perform the update on Windows and have the icon go away but on mac the icon just hangs around.

**Comment by IdrissDimson** [comment]
^Yeah, I recently got this issue too. A system to dismiss update icons would be ideal.

**Comment by alvint** [comment]
How has this not been fixed yet? Ridiculous.

**Comment by HarrisonRiddell** [comment]
It appears that this has been resolved? I'm not sure when since I've kind of just be ignoring it but I just opened Brackets and the update alert is gone.

**Comment by alvint** [comment]
Not resolved here.

**Comment by schroef** [comment]
Neither is it here

<img width="1038" alt="screen shot 2019-02-25 at 14 56 55" src="https://user-images.githubusercontent.com/6923008/53361067-9d6c7680-390d-11e9-971a-7a9c20838abd.png">
